Doc/library/sqlite3.rst

```diff
@@ -957,6 +968,20 @@ Connection Objects
 Cursor Objects
 --------------
 
+   A cursor object represents a database cursor,
+   which is used to execute SQL statements,
+   and manage the context of a fetch operation.
+   Cursors are created with :meth:`Connection.cursor`,
+   or by using any of the :ref:`connection shortcut methods
+   <sqlite3-connection-shortcuts>`.
+
+   Cursors objects are :term:`iterators <iterator>`,
+   meaning that if you :meth:`execute` a ``SELECT`` query,
```

Comment:
```suggestion
   meaning that if you :meth:`~Cursor.execute` a ``SELECT`` query,
```

You aren't inside the class here, so the reference is broken (and generates a warning)